README.md

```diff
@@ -1 +1,78 @@
-# drone-cobertura-s3
\ No newline at end of file
+# drone-cobertura-s3
+
+Drone plugin to upload cobertura code coverage reports to AWS S3 bucket and publish the bucket static site url to `Artifacts` tab under `Pipieline Execution`.
+
+## Build
+
+Build the binary with the following commands:
+
+```bash
+go build
+```
+
+## Docker
+
+Build the Docker image with the following commands:
+
+```
+./hacking/build.sh
```

Comment:
Why is the directory named hacking here? Please consider using a better name.

Reply:
Directory name changed to "scripts"